actionModel.json

```diff
@@ -0,0 +1,14 @@
+{
+    "properties": {
```

Comment:
Add : "schemaVersion": 1

for reference : https://github.com/OfficeDev/microsoft-teams-apps-survey/blob/main/actionModel.json

Reply:
Done